smelly_python/command_line.py

```diff
@@ -3,25 +3,32 @@
 """
 import sys
 import json
+import os
+import click
 from smelly_python.code_smell import CodeSmell
 from .generator.webpage_generator import generate_webpage
 
 
-def main():
+@click.command()
+@click.option('--file', help='Specify the json file generated by pylint.')
+@click.option('--command', help='The command to run pylint')
+def main(file, command):
     """
     Main command line interface.
     Takes the first command line argument to be the json generated by pylint.
     """
-    print(f'Argument count: {len(sys.argv)}')
-    if len(sys.argv) != 2:
-        print('Insufficient amount parameters provided')
+    if ((file is not None) and (command is not None) or (file == command)):
+        print("Please only provide either --file or --command argument")
         sys.exit(1)
-    with open(sys.argv[1], 'r', encoding='utf-8') as file:
-        dict_data = json.load(file)
+    file_name = file
+    if command:
+        print('Running pylint...')
+        print(command + ' --output-format=json > output.json')
+        os.system(command + ' --output-format=json > output.json')
+        file_name = './output.json'
```

Comment:
Don't think `./` is the way to go here, I think making an output folder might be better, or even a temp file but that might differ per os so scary.